src/containerapp-compose/azext_containerapp_compose/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.15.0",
+    "azext.maxCliCoreVersion": "2.36.0"
```

Comment:
Why do you need to set the `maxCliCoreVersion`?

Reply:
If you don't have the requirement for the `maxCliCoreVersion` of `azure-cli-core`, it is recommended to remove it. Otherwise, users using a CLI version newer than `2.36.0` version will not be able to install this extension